_posts/elements/2015-04-15-icons.md

```diff
@@ -0,0 +1,738 @@
+---
+layout:       element
+title:        Icons
+date:         2015-04-15 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- icons
+
+slug:         icons
+scssurl:      ""
+
+description:  ...
+
+info: ...
+---
+
+<h3 class="hd-6">Media</h3>
+<div class="example-set">
+    <div class="swatch">
+        <div class="icon-display">
+            <svg class="icon icon-video" title="User account"><use xlink:href="{{ site.baseurl }}public/pl/images/edx-svg/svgdefs.svg#icon-video"></use></svg>
```

Comment:
**Documentation Recommendation**: It would be to note what all of these elements and attributes do (as well as which are optional vs. required) in this syntax